app/main.py

```diff
@@ -1,3 +1,61 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as file:
+        config = json.load(file)
+
+    fuel_price = config["FUEL_PRICE"]
+    customers = []
+    shops = []
+
+    for config_customer in config["customers"]:
+        car = Car(
+            config_customer["car"]["brand"],
+            config_customer["car"]["fuel_consumption"]
+        )
+        customer = Customer(
+            config_customer["name"], config_customer["product_cart"],
+            config_customer["location"], config_customer["money"], car
+        )
+        customers.append(customer)
+
+    shops = [
+        Shop(config_shop["name"],
+             config_shop["location"],
+             config_shop["products"])
+        for config_shop in config["shops"]
+    ]
```

Comment:
```suggestion
    shops = [Shop(**config_shop) for config_shop in config["shops"]]
```
better to use unpacking in this case